src/test/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealServiceTest.java

```diff
@@ -518,4 +496,24 @@ public void getMatchedCases() {
 
         assertEquals(1, matchedCases.size());
     }
+
+    @Test
+    @Parameters({
+            "PIP, n1w1 wal, Birmingham",
+            "ESA, n1w1 wal, Birmingham",
+            "UC, n1w1 wal, Birmingham",
+            "PIP, NN85 1ss, Northampton",
+            "ESA, NN85 1ss, Northampton",
+            "UC, NN85 1ss, Northampton"
+
+    })
+    public void shouldSetProcessingVenueBasedOnBenefitTypeAndPostCode(String benefitCode, String postcode, String expectedVenue) {
```

Comment:
Could you add a test to cover the appointee route please?

Reply:
added